php/src/Controller/DockerController.php

```diff
@@ -191,8 +191,9 @@ public function StartContainer(Request $request, Response $response, array $args
         $config['install_latest_major'] = $installLatestMajor;
         $this->configurationManager->WriteConfig($config);
 
+        $pullImage = !isset($request->getParsedBody()['bypass_container_update']);
```

Comment:
```suggestion
        // Do not pull container images in case 'bypass_container_update' is set via url params
        // Needed for local testing
        $pullImage = !isset($request->getParsedBody()['bypass_container_update']);
        if ($pullImage === true) {
            error_log('WARNING: Not pulling container images. Instead, using local ones.');
        }
```